website/docs/r/user_provided_service.html.markdown

```diff
@@ -54,6 +54,7 @@ The following arguments are supported:
 * `credentials_json` - (Optional) Same as `credentials` but in the form of a stringified JSON object. Conflicts with `credentials` and defaults to empty map.
 * `syslog_drain_url` - (Optional) URL to which logs for bound applications will be streamed. Defaults to empty.
 * `route_service_url` - (Optional) URL to which requests for bound routes will be forwarded. Scheme for this URL must be https and defaults to empty
+* `recursive_delete` - (Optional, Bool) Default: `false`. If set `true`, Cloud Foundry will delete service bindings, service keys, and routes associated with the service instance too.
```

Comment:
```suggestion
```

There is no recursive delete support in the UPS delete api http://apidocs.cloudfoundry.org/4.4.0/user_provided_service_instances/delete_a_particular_user_provided_service_instance.html

Reply:
We targetted this API: http://apidocs.cloudfoundry.org/5.1.0/service_instances/delete_a_service_instance.html The latest version still supports the recursive flag.